package.json

```diff
@@ -10,16 +10,18 @@
   "author": "Thomas Jenkinson <tjcode@outlook.com>",
   "license": "MIT",
   "repository": "tjenkinson/clappr-thumbnails-plugin",
+  "dependencies": {
+    "clappr": "latest"
```

Comment:
I think actually Clappr might not be a dependency at all, as it's listed as an external in the webpack config?